english/eating-walls/find-a-toilet/door/pushed/pushed.md

```diff
@@ -0,0 +1,10 @@
+A tall thin man in jester's motley stands before you. He sighs and lights a cigarette.
+
+You look down and realize you have just wet your britches.  
+
+The man looks at you and riggles his fingers in the air like he's really spooky. "HAH! Skiddish are ya? Well, too late to show you to the restrooms..."
+
+You are slightly embarrassed but happy to not be holding it in anymore.
+
+The man takes a long drag from his cigarette, promptly turns you to face the open door, and [shoves you through the door](../lego-world/lego-world.md)

```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.